src/components/sdk/ContentShare/index.tsx

```diff
@@ -30,14 +70,25 @@ export const ContentShare: React.FC<Props> = ({ className, ...rest }) => {
     };
   }, [audioVideo, tileId]);
 
-  return tileId ? (
-    <ContentTile
-      objectFit="contain"
-      className={className || ''}
-      {...rest}
-      ref={videoEl}
-    />
-  ) : null;
+  return (
+    <>
+      <video
+        id="play-video"
```

Comment:
I don't think we need this ID since we're using refs, do we?

Reply:
Correct.